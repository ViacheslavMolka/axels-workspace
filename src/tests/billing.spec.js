import React from 'react';
import { shallow, configure } from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';

import BillingComponent from '../components/billing';


configure({ adapter: new Adapter() });

it('BillingComponent snapshot', () => {
    const component = shallow(<BillingComponent />);
    expect(component).toMatchSnapshot();
});
