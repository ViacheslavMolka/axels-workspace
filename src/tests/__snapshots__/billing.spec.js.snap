// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BillingComponent snapshot 1`] = `ShallowWrapper {}`;
